frontend/src/pages/projects/screens/spawner/environmentVariables/EnvTypeSelectField.tsx

```diff
@@ -36,6 +36,7 @@ const EnvTypeSelectField: React.FC<EnvTypeSelectFieldProps> = ({
               onToggle={() => setOpen(!open)}
               selections={envVariable.type || ''}
               placeholderText="Select one"
+              aria-label="Select one"
```

Comment:
Would this not be `Select environment variable type`? I get the UI doesn't say that -- not sure how much "help" we should grant to screen readers.

How much does an `aria-label` need to cover "description" of what is happening rather than echoing the text I assume that would be already read by the screen reader (I have not used a screen reader to test this).

Reply:
This fieldId of FormGroup needs to match the id of TextInput for VO to call out the form labels correctly! Agree that we could rename it to something that makes more sense - updated in https://github.com/opendatahub-io/odh-dashboard/commit/26c978e27e3bbcae56e254a0180567126c06165c#diff-0dc485737af4aa9ff1e7bc1cb83ff242afc309d6a92b34362c5cbaf61b855b26

And that is a good question - this may be related to an a11y bug within the `Select` component in PF. Waiting back on the PF team to confirm...

Reply:
The text should be updated upon rebasing https://github.com/opendatahub-io/odh-dashboard/pull/994/files. I tested with screenreader, and it seems like PF has some issues with the current select component and the aria-label/placeholder text doesn't get called out by VO. However this will be resolved in the next major release as PF is introducing a new Select component that has the correct VO behavior @andrewballantyne